code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -223,5 +196,128 @@
 			//Prevents whatever mechanism is causing it to hit negative temperatures.
 		return REACTING
 
+/datum/gas_reaction/brownsformation //The formation of brown gas. Endothermic.
+	priority = 3
+	name = "Brown Gas formation"
+	id = "brownsformation"
+
+/datum/gas_reaction/brownsformation/init_reqs()
+	min_requirements = list(
+		"oxygen" = 20,
+		"nitrogen" = 20,
+		"temp" = FIRE_MINIMUM_TEMPERATURE_TO_EXIST*200
+	)
+
+/datum/gas_reaction/brownsformation/react(datum/gas_mixture/air)
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+
+	var/old_heat_capacity = air.heat_capacity()
+	var/heat_efficency = temperature/FIRE_MINIMUM_TEMPERATURE_TO_EXIST*100
+	var/energy_used = heat_efficency*BROWNS_FORMATION_ENERGY
+	air.assert_gases("oxygen","nitrogen","browns")
```

Comment:
nice trick here; because o2 and n2 have minimum requirements, you needn't assert them